examples/erc20-counter/deployment-guide.md

```diff
@@ -73,86 +77,67 @@ You can deploy your contracts and run an end-to-end test or demo as follows:
     > export COUNTER_ADDRESS=$(jq -re '.transactions[] | select(.contractName == "Counter") | .contractAddress' ./broadcast/DeployCounter.s.sol/31337/run-latest.json)
     > ```
 
-5. Mint some Toyken:
-    ```bash
-    cast send --private-key $ETH_WALLET_PRIVATE_KEY --rpc-url http://localhost:8545 $TOYKEN_ADDRESS 'mint(address, uint256)' 0x9737100D2F42a196DE56ED0d1f6fF598a250E7E4 100
-    ```
-    > Now the account at address `0x9737100D2F42a196DE56ED0d1f6fF598a250E7E4` should have 100 Toyken.
-    >
-    > You can verify the balance by running:
-    > ```bash
-    > cast call $TOYKEN_ADDRESS --rpc-url http://localhost:8545 "balanceOf(address)(uint256)" 0x9737100D2F42a196DE56ED0d1f6fF598a250E7E4
-    > ```
-
 ### Interact with your local deployment
 
 1. Query the state:
 
     ```bash
-    cast call --rpc-url http://localhost:8545 ${COUNTER_ADDRESS:?} 'get()(uint256)'
+    cast call --rpc-url http://localhost:8545 $COUNTER_ADDRESS 'get()(uint256)'
```

Comment:
I only kept the `${_:?}` when using external variables. Otherwise, in the case of `COUNTER_ADDRESS`, if someone followed the deployment guide, they had an `export COUNTER_ADDRESS` anyway, so this variable will always exist.
I think $COUNTER_ADDRESS is a bit more readable and works on non-bash shells, so I decided to go with this compromise, but I'm happy to change this back. I don't have a strong opinion on this.

Reply:
It is more readable, indeed. Let's go with this change, thank 👍